assets/css/style.css

```diff
@@ -1,276 +1,398 @@
+  background-repeat: no-repeat;
+  border-bottom-left-radius: 5rem;
+  border-top-left-radius: 2rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+nav {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding: 1.6rem 5rem;
+  background-color: white;
+  width: 100%;
+  position: fixed;
+  height: 46px;
+  top: 0;
+  left: 0;
 }
 
-.logo {
-  text-decoration: none;
-  color: var(--color-accent);
-  font-size: 1.8rem;
-  font-weight: 700;
+.nav-desk {
+  display: none;
 }
 
-.fa-bars {
-  color: var(--color-accent);
-  font-size: 16px;
+.nav-desk-ul {
+  display: flex;
+}
+
+.nav-desk-li a {
+  padding-right: 32px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  color: #344563;
+}
+
+header {
+  background: white;
+  position: fixed;
+  z-index: 10;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  padding-left: 10px;
 }
 
-.headline-container {
+.headline {
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  align-items: flex-start;
-  max-width: 100rem;
-  margin: 0 auto;
-  margin-top: 3rem;
-  top: 15%;
-  padding: 2.5rem;
-  position: absolute;
-}
-
-.headline-heading {
-  font-size: 4rem;
-  color: var(--color-headings);
-  line-height: 1.1;
-  width: 80%;
-  margin-top: 4rem;
-}
-
-.headline-text {
-  color: var(--color-headings);
-  margin-top: 1rem;
-  padding-right: 1.4rem;
-  font-size: 12px;
-  line-height: 24px;
+  position: relative;
+  padding: 6rem 1.5rem;
+}
+
+.headline h1 {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 800;
+  color: #172b4d;
+}
+
+.headline p {
+  font-size: 16px;
   font-weight: 400;
-  justify-content: center;
+  line-height: 24px;
+  color: #344563;
 }
 
-.connect-text {
-  font-weight: 500;
+.connect,
+.connect:visited,
+.connect:hover,
+.connect:active {
   color: #7f8cff;
-  margin-top: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
 }
 
-.headline-icons {
-  margin-top: 0.5rem;
-  list-style: none;
+.brands {
+  margin: 15px 0 15px -5px;
   display: flex;
+  list-style: none;
+}
+
+.brands i {
+  padding-left: 10px;
+  color: #505f79;
+  font-size: 20px;
+  font-style: normal;
+}
+
+.button-mobile {
+  margin-right: 15px;
+}
+
+.desktop-img {
+  display: none;
 }
 
-.headline-icon {
-  padding: 0.7rem;
+.mobile-img {
+  display: block;
 }
 
-.card {
+.works-portfolio {
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  background: var(--white);
-  border-radius: 20px;
-  margin-bottom: 10rem;
-  padding-bottom: 2rem;
-  width: 80vw;
+  justify-content: center;
 }
 
-.card-image {
+.work-inside {
+  display: grid;
+  grid-template-rows: repeat(3, auto);
   width: 90%;
-  margin-top: 1.5rem;
+  list-style: none;
+  padding: 8rem 1.6rem;
+  row-gap: 5em;
+  justify-content: center;
```

Comment:
- [ ] Please add `margin: 0 auto;` after your width property to center the work cards to match the Figma design.
- [ ] Please increase the `row-gap` value to match the Figma design and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/167131403-ade6f460-5b0a-4259-b615-40255a729dcc.png) | ![image](https://user-images.githubusercontent.com/68796232/167131385-31dace70-282d-4375-ba2b-65a4b15511f5.png)  |